Assets/Scripts/Buddy/BuddyStats.cs

```diff
@@ -4,11 +4,14 @@
 public class BuddyStats : MonoBehaviour
 {
 	public Stat statType = Stat.Invalid;
-	[SerializeField] int startingApples = 10;
-	[SerializeField] int apples = 0;
+	[SerializeField] bool canDecreaseStat = true;
+	[SerializeField] int startingResources = 10;
```

Comment:
A more descriptive/concise name for this might be initResourceCount or startingResourceCount. This could be misconstrued as and array of resources prefabs as is